service/src/test/java/uk/gov/hmcts/reform/fpl/service/TaskListRendererTest.java

```diff
@@ -67,20 +61,8 @@
 
     @Test
     void shouldRenderTaskListWhenApplicationDocumentsIsToggledOff() {
-        given(featureToggleService.isApplicationDocumentsEventEnabled()).willReturn(false);
-
         final String expectedTaskList = readString("task-list/expected-task-list.md").trim();
 
         assertThat(taskListRenderer.render(getTasks(DOCUMENTS))).isEqualTo(expectedTaskList);
     }
-
-    @Test
-    void shouldRenderTaskListWhenApplicationDocumentsIsToggledOn() {
-        given(featureToggleService.isApplicationDocumentsEventEnabled()).willReturn(true);
```

Comment:
this is the one that should be kept as it references getTasks(APPLICATION_DOCUMENTS) not DOCUMENTS but task-list/expected-task-list-when-application-documents-enabled.md can be renamed now to task-list/expected-task-list.md and replace old one